submissionforms.md

```diff
@@ -30,6 +30,7 @@ Provide detailed information about a specific input-form. The JSON response docu
           "mandatory": false,
           "repeatable": true,
           "hints": "Enter the names of the authors of this item.",
+          "nameVariants": true,
```

Comment:
Just to clarify: Would you like to move this setting inside the `selectableRelationship` sections in the submission form configuration?

Reply:
Yes @AlexanderS , that makes more sense to me.

Reply:
Yes @AlexanderS , that makes more sense to me.

Reply:
It's possible to move it to the `selectableRelationship` section, but do keep in mind that if it would be moved, it would not be possible to use it for name variants in e.g. DSpace 8.
By making it a property of the entire field, it would be much easier to extend this functionality in DSpace 8

Reply:
@benbosman do you think we should brought this into discussion for having a broad agreement on this? As far as DSpace 8, there are several open points of discussion, and perhaps, this solution might fit in Relationships Metadata and Relationships as DSpaceObjects.

Reply:
This was discussed yesterday, and the preference was to support this only for entities.
I've moved the parameter in the last commit and it is now consistent with the suggestions in this thread